vaadin-radio-button-flow-parent/vaadin-radio-button-flow/src/main/java/com/vaadin/flow/component/radiobutton/FieldValidationUtil.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2000-2020 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.radiobutton;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.HasValidation;
+import com.vaadin.flow.component.page.PendingJavaScriptResult;
+
+class FieldValidationUtil {
+    private FieldValidationUtil() {
+
+    }
+
+    static void disableClientValidation(Component component) {
```

Comment:
Is this called anywhere other than the RadioButtonGroup constructor? If not, would it make sense to include this code in the constructor or as a private method instead of a new class? This would simplify the code, as you wouldn't need to check if `component instanceof HasValidation`.